Lib/python/pyrun.swg

```diff
@@ -343,6 +355,9 @@ SwigPyObject_get___dict__(PyObject *v, PyObject *SWIGUNUSEDPARM(args))
   if (!sobj->dict)
     sobj->dict = PyDict_New();
 
+  if (!sobj->dict)
```

Comment:
Similarly, this could be solved by `Py_XINCREF()` below, since `PyDict_New` sets up the OOM exception. Would you prefer that as a change?

Reply:
Yes please.